data/tlg0007/tlg071/tlg0007.tlg071.perseus-eng3.xml

```diff
@@ -0,0 +1,808 @@
+         </titleStmt>
+         <!--<extent/> &Perseus.publish;-->
+      	<publicationStmt>
+      		<publisher>Trustees of Tufts University</publisher>
+      		<pubPlace>Medford, MA</pubPlace>
+      		<authority>Perseus Project</authority>
+      		<date type="release">2010-12-13</date>
+      	</publicationStmt>
+      	<sourceDesc>
+            <biblStruct>
+               <!--<analytic/>-->
+               <monogr>
+                  <author>Plutarch</author>
+                  <title rend="italic" xml:lang="eng">Moralia</title>
+                  <respStmt>
+                     <resp>with an English Translation by</resp>
+                     <name>Frank Cole Babbitt</name>
+                  </respStmt>
+                  <imprint>
+                     <pubPlace>Cambridge, MA</pubPlace>
+                     <publisher>Harvard University Press</publisher>
+                     <pubPlace>London</pubPlace>
+                     <publisher>William Heinemann Ltd.</publisher>
+                     <date>1927</date>
+                  </imprint>
+                  <biblScope unit="volume">1</biblScope>
+               </monogr>
+            	<ref target="https://archive.org/details/in.ernet.dli.2015.43837/page/n441">The Internet Archive</ref>
+            </biblStruct>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <!--<editorialDecl>
+            <correction status="high" method="silent">
+               <p>optical character recognition</p>
+            </correction>
+         </editorialDecl>-->
+      	<p>Text encoded in accordance with the latest EpiDoc standards</p> 
+      	<p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture</p>    
+         <refsDecl n="CTS">
+         	<cRefPattern n="section" matchPattern="(\w+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+         		<p>This pointer pattern extracts sections</p></cRefPattern>
+            <!--<state unit="section"/>-->
+         </refsDecl><!--<refsDecl doctype="TEI.2">
+				<state unit="stephpage" n="chunk"/>
+				</refsDecl>--></encodingDesc>
+      <profileDesc>
+         <langUsage>
+         	<language ident="eng">English</language>
+         	<language ident="grc">Greek</language>
+         	<language ident="lat">Latin</language>
+         	<language ident="deu">German</language>
+         </langUsage>
+      </profileDesc>
+      <revisionDesc>
+      	<change who="RS" when="2010-05">tagged and parsed</change>
+      	<change who="Sophia Elzie" when="2019-06-21">CTS and Epidoc conversion</change>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="eng">
+      <body>
+      	<div type="translation" xml:lang="eng" n="urn:cts:greekLit:tlg0007.tlg071.perseus-eng3">
+         <pb xml:id="v1.p.399"/>
+         <div type="textpart" subtype="section">
+            <head>HOW A MAN MAY BECOME AWARE OF HIS PROGRESS IN VIRTUE (QUOMODO QUIS SUOS IN
```

Comment:
I just spotted this in another file. Please tag the Latin as such and close the head after the Latin parentheses. Then Introduction should be something like <head type="subhead">INTRODUCTION</head> (it could also be <label> or something similar. No hard and fast rule here.)
I just want to distinguish the work header from the section subhead.

Reply:
Running through Travis now, what I did here was move the header outside of the intro div and moved Introduction inside the paragraph.